Server/Program.cs

```diff
@@ -205,13 +205,16 @@
 services.AddScoped(x => (CircuitHandler)x.GetRequiredService<ICircuitConnection>());
 services.AddSingleton<ICircuitManager, CircuitManager>();
 services.AddScoped<IAuthService, AuthService>();
-services.AddScoped<IClientAppState, ClientAppState>();
+services.AddScoped<ISelectedCardsStore, SelectedCardsStore>();
 services.AddScoped<IExpiringTokenService, ExpiringTokenService>();
 services.AddScoped<IScriptScheduleDispatcher, ScriptScheduleDispatcher>();
 services.AddSingleton<IOtpProvider, OtpProvider>();
 services.AddSingleton<IEmbeddedServerDataSearcher, EmbeddedServerDataSearcher>();
 services.AddSingleton<ILogsManager, LogsManager>();
-services.AddScoped<IMessenger>((services) => new WeakReferenceMessenger());
+services.AddScoped<IThemeProvider, ThemeProvider>();
+services.AddScoped<IChatSessionStore, ChatSessionStore>();
```

Comment:
Since we're adding these as scoped, wouldn't that mean if I refresh the page I'd lose my chats?  Is that how it behaved before?

Reply:
Yeah, it would.  That's how it behaved before.